apps/condo/domains/common/components/Button.tsx

```diff
@@ -275,9 +276,14 @@ const BUTTON_TYPE_STYLES = {
     sberBlack: sberBlackCss,
 }
 
-export const Button: React.FC<CustomButtonProps> = ({ type, secondary, ...restProps }) => {
+export const Button: React.FC<CustomButtonProps> = (props) => {
+    const { type, secondary, eventName, eventProperties, ...restProps } = props
+    const { instrument } = useTracking()
+
+    const onClick = eventName ? instrument(eventName, eventProperties, restProps.onClick) : restProps.onClick
+
     if (!SKIP_BUTTON_TYPES_FOR_DEFAULT.includes(type)) {
-        return <DefaultButton {...restProps} type={type as ButtonProps['type']}/>
+        return <DefaultButton {...restProps} type={type as ButtonProps['type']} onClick={onClick}/>
```

Comment:
you need to delete onClieck from restProps

Reply:
Done